worldedit-sponge/build.gradle

```diff
@@ -0,0 +1,90 @@
+apply plugin: 'eclipse'
+apply plugin: 'idea'
+
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { url = "http://files.minecraftforge.net/maven" }
+        maven { url = "https://oss.sonatype.org/content/repositories/snapshots/" }
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'net.minecraftforge.gradle:ForgeGradle:2.1-SNAPSHOT'
+    }
+}
+
+apply plugin: 'net.minecraftforge.gradle.forge'
```

Comment:
SpongeVanilla uses exactly the same SRG/MCP environment at runtime, just with the little difference that a) it will always be only on the server side and b) Forge modifications or additions to Minecraft classes are missing. From what I can tell, almost everything here is just Vanilla, but it's better to use a Vanilla workspace directly so you notice the usage of Forge methods directly.

Except very few places where Forge actually moves logic or modifies existing methods the code will always work on both platforms. It would be essentially exactly the same as the workspace used by SpongeCommon.

There is an example for a Vanilla server workspace at https://github.com/Minecrell/worldedit-spongevanilla/blob/spongevanilla/worldedit-forge/build.gradle